elasticsearch-agent/src/main/resources/application.properties

```diff
@@ -3,21 +3,21 @@ server.port=8080
 server.servlet.context-path=/elastic-agent
 
 #ElasticSearch Client Configuration
-elasticsearch.client.url=localhost
-elasticsearch.client.port=9200
+elasticsearch.client.url=${ES_CLIENT_URL}
```

Comment:
may be we should add default values for all properties where it is possible?

Reply:
Done.